common/rfb/CMsgWriter.cxx

```diff
@@ -204,6 +204,25 @@ void CMsgWriter::writeClientCutText(const char* str)
   endMsg();
 }
 
+void CMsgWriter::writeQemuAudioEnableOrDisable(bool enable)
+{
+  startMsg(msgTypeQEMUClientMessage);
+  os->writeU8(qemuAudio);
+  os->writeU16(enable ? 0 : 1);
+  endMsg();
+}
+
+void CMsgWriter::writeQemuAudioSetFormat(rdr::U8 fmt, rdr::U8 channels, rdr::U32 frequency)
+{
+  startMsg(msgTypeQEMUClientMessage);
+  os->writeU8(qemuAudio);
+  os->writeU16(2);
```

Comment:
Please try to define constants for things such as this. Makes it easier to read this code.

Reply:
OK